src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/29.diff.sql

```diff
@@ -0,0 +1,91 @@
+CREATE INDEX IX_Status ON dbo.TaskInfo (Status)
```

Comment:
create or alert index?

Reply:
Alter is not allowed for index, use a if to determine the existence of the Index.